### ISSUE TITLE
Problem with 1.4.0 version of dbplyr

### ISSUE THREAD

**Issue by robkooper** [PecanProject/pecan#2349]
### Bug Description

When using version 1.4.0 of dbplyr there is an issue where travis builds are failing. Installing version 1.3.0 of dbplyr fixes. We should figure out what can be done so we can work with the latest version of dbplyr.

This could be related to https://github.com/tidyverse/dbplyr/issues/269

**Comment by ashiklom** [comment]
One workaround that may end up being necessary (in fact, the `dplyr` folks may decide this is a _feature_) is to unquote (`!!`) all of our queries. This unambiguously forces local evaluation, rather than `dplyr`'s attempts to translate to SQL.

In other words, code like this:

```r
my_var <- list(var_id = 81)
tbl(con, "traits") %>%
  filter(variable_id == my_var$var_id)
```

...should become:

```r
my_var <- list(var_id = 81)
tbl(con, "traits") %>%
  filter(variable_id == !!my_var$var_id)
```

I _think_ code like this should still work...

```r
my_var_id <- 81
tbl(con, "traits") %>%
  filter(variable_id == my_var_id)
```

...but just to be safe, it may be worth unquoting this too, to be completely unambiguous:

```r
my_var_id <- 81
tbl(con, "traits" %>%
  filter(variable_id == !!my_var_id)
```

**Comment by ashiklom** [comment]
I'm about 75% sure this has been resolved by #2392. But somebody should double-check.
